### DISCUSSION TITLE
Disabled actions on index view

### DISCUSSION THREAD

**Discussion by oleksandr-storchak** [avo-hq/avo#1006] [Help] [answered]
Why i can't use actions on index view of resource? 

<img width="886" alt="Screenshot 2022-06-30 at 17 49 06" src="https://user-images.githubusercontent.com/22897472/176708889-bb99560c-ed53-485f-9f11-eb981ef0319b.png">
<img width="731" alt="Screenshot 2022-06-30 at 17 50 41" src="https://user-images.githubusercontent.com/22897472/176708905-77cfa8c8-5df6-4ac8-a362-c2f994e762a5.png">

**Comment by adrianthedev** [comment, marked as answer]
They are disabled by default. You have to select a record on the table to enable them.

**Reply by oleksandr-storchak** [reply]
Thanks for answer! I can enable it for general actions?

**Reply by adrianthedev** [reply]
Yes. They are called `Standalone actions`.

https://docs.avohq.io/2.0/actions.html#standalone-actions

You can also refer to the docs in the future. They are pretty complete 💪 and they'll give you a lot of guidance.

**Reply by oleksandr-storchak** [reply]
Thank you ! Off course i use documentation, but in this case i completely forgot about standalone-actions

**Reply by adrianthedev** [reply]
No worries man! Just wanted to solidify the fact that the docs are there and contain a lot of information.

Thanks!
